cookiecutter.json

```diff
@@ -1,15 +1,18 @@
 {
-  "app_name": "template",
-  "app_title": "Application template",
-  "short_description": "The default output of the AiiDAlab application cutter.",
-  "github_user": "aiidalab",
-  "repo_name": "aiidalab-{{cookiecutter.app_name}}",
+  "app_name": "my-app",
```

Comment:
having a dash in the default name could make the user wonder whether they should write `aiidalab-...` here.
anyhow, that's a very minor comment

Reply:
@ltalirz This is actually related to a slight inconsistency that I discovered. The app name is recommended to be hyphen-delimited without the `aiidalab-` prefix, while the repo name should have the prefix. This is how we generate the default repo-name currently. However, this means that the repo is put into `apps/` with `apps/aiidalab-my-app`, but we actually only want it to be `apps/my-app`, because that's how it would be installed from the registry if registered as `my-app`.

One way to resolve this problem would be to simply remove the recommendation to prefix app repositories with `aiidalab-`. What do you think?

Reply:
Another option would be to simply distinguish between the local "app folder name" and the "remote repository name". That might actually be an elegant solution. I can create a PR for that if you like.

Reply:
Third option: use `aiidalab-home`, .. as the folder name.
I think that would be the canonical solution (if you think of the analogue with python packages).



> One way to resolve this problem would be to simply remove the recommendation to prefix app repositories with `aiidalab-`. What do you think?

I would advise to keep the prefix since it makes it very easy to identify whether a git repository contains an app or is aiidalab-related (to sort + to search for it)